implement handle_submit to send scores to the web

awli opened the pull request.

awli committed:
implement handle_submit to send scores to the web

awli committed:
send actual score values instead of sending them all in autonomous

awli committed:
fixed something broken

nikitakit commented on src/scheduler.py:
```diff
@@ -129,18 +132,59 @@ def handle_submit(self, channel, data):
+            "number" : 1, 
+            "autonomous" : score_state.blue_autonomous_points, 
+            "bonus" : bonus_points[0], 
+            "manual" : score_state.blue_normal_points, 
```
this is entirely missing the permanent goal points

nikitakit commented on src/scheduler.py:
```diff
@@ -129,18 +132,59 @@ def handle_submit(self, channel, data):
+            "number" : 2, 
+            "autonomous" : score_state.gold_autonomous_points, 
+            "bonus" : bonus_points[1], 
+            "manual" : score_state.gold_normal_points, 
```
Same here: no perma goal points anywhere

awli committed:
include permanent points in points

awli committed:
force https on match upload

nikitakit commented on src/scheduler.py:
```diff
@@ -129,18 +131,59 @@ def handle_submit(self, channel, data):
+        # TODO: Put the score determining logic in handle_score
+        score_state = self.score_states[match_num]
+        bonus_points = [0, 0]
+        if score_state.bonus_possession == forseti2.score_state.BLUE:
```
possession is defined as the bonus ball being on your side of the field.

nikitakit committed:
Fix bonus ball score submission

nikitakit committed:
Remove extra field in score submision

The "number" field is the POST request is not used, so it should be removed to avoid confusion.
